src/ports/SkFontMgr_win_dw.cpp

```diff
@@ -93,9 +95,14 @@ SK_STDMETHODIMP StreamFontFileLoader::CreateStreamFromKey(
     UINT32 fontFileReferenceKeySize,
     IDWriteFontFileStream** fontFileStream)
 {
+    SkRecordReplayAssert("[RUN-2058] StreamFontFileLoader::CreateStreamFromKey");
```

Comment:
(Tangential: Chris cooked up a `SkopedAssert` for these types of situations - I am seeing its use-case more and more often 🤔 )